app/commands/user/reputation_token/award_for_pull_request.rb

```diff
@@ -26,6 +26,7 @@ def award_reputation_to_author
           level: author_reputation_level,
           repo: repo,
           pr_id: pr_id,
+          pr_number: pr_number,
```

Comment:
We used to use the PR's number as its ID, but that was actually not the best option, as the number is only unique in combination with the repository. As explained elsewhere, we are now using a real, proper ID for pull requests.